### PR TITLE
Style checkout dialogs, fix several bugs, and add various other Colin Tweaks™

### PR DESCRIPTION
**Pull request by Anaphase**
I suggest going through the commits one-by-one on this one instead of looking all all the changes together!

### CODE REVIEW

**Comment by Anaphase** [comment]
screen shot 2018-12-04 at 5 23 35 pm

File: src/components/CODXCheckoutControl.vue [hunk 1]
```diff
@@ -4,30 +4,43 @@
...
+        <div class="line-item" :class="{ 'insufficient-codx': insufficientCODX }">
+          <span>Current Balance</span>
+          <span class="current-balance">{{ user.codxBalance | formatCODXAmount }}</span>
+        </div>
 
-      <div v-if="insufficientCODX">
-        <b-button to="/get-codx">
-          Buy more CODX
-        </b-button>
+        <div class="insufficient-codx-notice"  v-if="insufficientCODX">
```
Comment (johnhforrest) [inline src/components/CODXCheckoutControl.vue]:
```suggestion
        <div class="insufficient-codx-notice" v-if="insufficientCODX">
```

Remove extra space

File: src/components/modals/MetaMaskNotificationModal.vue [hunk 2]
```diff
@@ -201,16 +193,23 @@ export default {
       if (typeof this.onClear === 'function') this.onClear()
     },
 
+    scrollToTop() {
+      // annoyling enough, the element that Vue Bootstrap attaches the b-modal
```
Comment (johnhforrest) [inline src/components/modals/MetaMaskNotificationModal.vue]:
```suggestion
      // annoyingly enough, the element that Vue Bootstrap attaches the b-modal
```

Fix spelling

**Review by johnhforrest — APPROVED** [review]
Fixed some minor things inline

**Pull request merged**